### DISCUSSION TITLE
uac.exe doesn't close

### DISCUSSION THREAD

**Discussion by ramonbra** [Eugeny/tabby#8243] [Q&A]
Hi everyone! 
Just started using Tabby and I managed to run consoles as admin by default. The problem is, whenever a new Terminal is opened (I use PowerShell, if it matters) uac.exe opens, but when I close the terminal uac.exe stays open. I don't know if there's something in config that would help with that but I can't seem to be able to solve the problem. 

TL;DR: Does someone know if uac.exe can be closed on terminal exit?

**Comment by arthur-ermolaev** [comment]
@ramonbra Did you find a resolution?
